src/main/cli/cli.c

```diff
@@ -4962,8 +4962,8 @@ static void cliRcSmoothing(const char *cmdName, char *cmdline)
             cliPrintLine("(manual)");
         }
         cliPrintf("# Derivative filter type: %s", lookupTables[TABLE_RC_SMOOTHING_DERIVATIVE_TYPE].values[rcSmoothingData->derivativeFilterType]);
-        if (rcSmoothingData->derivativeFilterTypeSetting == RC_SMOOTHING_DERIVATIVE_AUTO) {
-            cliPrintLine(" (auto)");
+        if (rcSmoothingData->derivativeFilterTypeSetting == RC_SMOOTHING_DERIVATIVE_BIQUAD) {
```

Comment:
This isn't needed - the `cliPrintf()` line above will already print 'biquad' in this case.

Reply:
Good, I see.
Thanks.
Fixed